generator/source/faq/videos.rst

```diff
@@ -31,6 +31,12 @@ Intel: Building Blockchain for the Enterprise. Shows how Sawtooth builds are don
 Sawtooth 1.0 Architecture and App Development (31:26, Zac Delventhal, Bitwise, 2018)
     https://www.youtube.com/watch?v=uBebFQM49Xk
 
+Hyperledger Sawtooth Technical Overview Development (58:50, Dan Anderson, Intel, 2018)
+    https://www.youtube.com/watch?v=KGFuKIBma_M
+
+Hyperledger Sawtooth Application Development (55:16, Dan Anderson, Intel, 2018)
```

Comment:
This is the "text title" of the youtube video, but I prefer the title shown in the actual video: Hyperledger Sawtooth Application Developer Overview using "CookieJar" (it's more descriptive).